.github/workflows/PR-U20.yaml

```diff
@@ -0,0 +1,127 @@
+name: OpenCV U20
+
+on:
+  pull_request:
+    branches:
+      - 3.4
+      - 4.x
+      - 5.x
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DBUILD_opencv_xfeatures2d=OFF -DENABLE_CCACHE=OFF'
```

Comment:
I propose to add `ENABLE_NONFREE` and test non-free algorithms too. Why do you exclude xfeatures2d?

Reply:
Okay, I'll add `ENABLE_NONFREE` here. I added `-DBUILD_opencv_xfeatures2d=OFF` flag, because the idea was to re-use Jenkins pipelines and this flag was right there. I'll remove it.

Reply:
Added `ENABLE_NONFREE` flag for cmake.